spec/models/post.spec.js

```diff
@@ -10,11 +10,16 @@ describe("Post model", () => {
     });
   });
 
-  it("has a message", () => {
+  it("post has a message", () => {
     var post = new Post({ message: "some message" });
     expect(post.message).toEqual("some message");
   });
 
+  it("post has a date", () => {
+    var post = new Post({ date: Date().slice(0, -31)});
+    expect(post.date).toEqual( Date().slice(0, -31) );
+  });
+
```

Comment:
looks good 
I think in theory it would be good to assign a mocked date (as currently the result will change) but I can see that this will pass, at least mostly, as the test will be run in seconds